lake-manifest.json

```diff
@@ -20,7 +20,7 @@
   {"git":
    {"url": "https://github.com/leanprover-community/aesop",
     "subDir?": null,
-    "rev": "b601328752091a1cfcaebdd6b6b7c30dc5ffd946",
+    "rev": "9dc4a1097a690216eaa7cf2d2290efd447e60d7a",
```

Comment:
Is this intended? Is this a bump or a downgrade?

Reply:
Yes, I ran `lake update` again.